client/layout/masterbar/checkout.tsx

```diff
@@ -108,7 +125,21 @@ const CheckoutMasterbar = ( {
 				<span className="masterbar__secure-checkout-text">{ translate( 'Secure checkout' ) }</span>
 			</div>
 			{ title && <Item className="masterbar__item-title">{ title }</Item> }
-			{ loadHelpCenterIcon && <DefaultMasterbarContact /> }
+
+			<div className="masterbar__item-help-icons">
+				{ loadHelpCenterIcon && <DefaultMasterbarContact /> }
+
+				<ChatButton
+					chatIntent="PRESALES"
+					initialMessage={ initialMessage }
+					siteUrl={ siteSlug }
+					className={ classnames( 'masterbar__item' ) }
```

Comment:
Minor, but you don't need to use the `classnames()` helper if you don't have dynamic class names.
```suggestion
					className="masterbar__item"
```